recipes/virtualenv/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "virtualenv" %}
+{% set version = "15.1.0" %}
+{% set sha256 = "aea627d114a3863d6374c5a3fc3cdd08907e0ac951cf93b458e5ba5998c516de" %}
+
+package:
+  name: "{{ name }}"
+  version: "{{ version }}"
+
+source:
+  fn: "{{ version }}.tar.gz"
+  url: https://github.com/pypa/virtualenv/archive/{{ version }}.tar.gz
+  sha256: "{{ sha256 }}"
+  patches:
+    - 0001-Mach-O-Copy-any-dynamic-libraries-and-symlinks-to-th.patch
+    - 0002-ELF-Copy-any-dynamic-libraries-and-symlinks-to-the-c.patch
+
+build:
+  number: 0
+  entry_points:
+    - virtualenv = virtualenv:main
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+
+test:
+  commands:
+    - virtualenv --help
+    - |
+      virtualenv -p "$(which python)" /tmp/venv-$$                                # [not win]
+      source /tmp/venv-$$/bin/activate                                            # [not win]
+      python -c 'import sys ; sys.exit(not hasattr(sys, "real_prefix"))'          # [not win]
```

Comment:
I'm not sure virtualenv was ever even a problem on windows, and regardless I don't think this PR changes virtualenv's behavior on Windows.